src/scripts/heroAnimation/__tests__/utils.test.js

```diff
@@ -134,12 +134,21 @@ describe("getDistance function", () => {
   });
 
   it("should use sqrt method to calculate distance between 2 points using sqrt as fallback", () => {
-    global.Math.hypot = undefined;
-    global.Math.sqrt = jest.fn();
+    const backupHypot = global.Math.hypot;
```

Comment:
@danbull @ThePeach I updated this test because I found a bug by mistake. If you changed the order of the tests (first sqrt, and then hypot)  on the previous version you will see the error. I think the reason was that I was manipulating the Math Object for the test, but I didn't restore it for the future tests.